audit-service/pom.xml

```diff
@@ -306,6 +306,44 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Need one standard for all of the services.  Expect that we are going to use the attribute-masking-service as the template so the related Maven pom file should be used to modify the audit-service pom file.  Details will need to include moving the test configuration properties out of this file.

Reply:
This was added since tests were in the new style just so they would run.
Now we have everything moved over, we should be using the root pom.

I believe there is a ticket in the backlog for analyzing dependencies, but I will check